reviews/templates/reviews/review_table.html

```diff
@@ -17,7 +17,7 @@
                 <td>{{ decision.reviewer.get_full_name }}</td>
                 <td>{{ decision.get_go_display|default:_("open") }}</td>
                 <td data-order="{{ decision.date_decision|date:'c' }}">{{ decision.date_decision|date:"j F Y, G:i" }}</td>
-                <td>{{ decision.comments }}</td>
+                <td>{{ decision.comments | linebreaks }}</td>
```

Comment:
I never knew about this filter. Now I feel like an idiot for `<br />`ifying everything by hand in grant-tool. Good find